android/expoview/build.gradle

```diff
@@ -677,6 +677,7 @@ task cleanNdkLibs(type: Exec) {
 task packageNdkLibs(dependsOn: buildReanimated, type: Copy) {
   from("$buildDir/reanimated-ndk/all")
   include("**/lib*reanimated*.so")
+  include("**/lib*turbomodulejsijni*.so")
```

Comment:
What is supposed to be inside that library?

Reply:
reanimated and turbomodulejsijni

Reply:
yes I meant turbomodulejsijni, what is supposed to be there?